### PR TITLE
Changed to real addresses and modified vendor names

### PR DESCRIPTION
**Pull request by Wenhuan2516**
I noticed that if the latitude and longitude information were not real, the vendors would be located on the ocean easily. I modified them to be real addresses with real latitude and longitude.

### CODE REVIEW

File: backend/vendors.json [hunk 1]
```diff
@@ -0,0 +1,206 @@
+{ "index": { "_index": "vendors", "_type": "vendor", "_id": "0" } }
```
Comment (MakotoE) [inline backend/vendors.json:1]: Please remove this file as it is not necessary.

**Review by MakotoE — CHANGES_REQUESTED** [review]
You're missing the table schemas in reset_database for `Areas` and `CuisineTypes`.

File: backend/vendors.json (hunk 1 above)
Comment (Wenhuan2516) [reply backend/vendors.json]: Got it. Already added these two tables to the schema and deleted the first piece of data in JSON

File: backend/database/database.go [hunk 2]
```diff
@@ -706,3 +706,89 @@ func (d *Database) StarDelete(userID uuid.UUID, vendorID uuid.UUID) error {
...
+type CuisineTypes struct {
+	VendorID    uuid.UUID
+	CuisineType string
+}
+
+func (d *Database) CuisineTypesCreate(cuisineType *CuisineTypes) error {
+	const command = `
+		INSERT INTO CuisineTypes (
+			VendorID,
+			CuisineType,
```
Comment (MakotoE) [inline backend/database/database.go:762]: You have an extra comma here.

**Review by MakotoE — CHANGES_REQUESTED** [review]

File: backend/api.go [hunk 3]
```diff
@@ -731,3 +736,120 @@ func (a *API) StarsDelete(c *gin.Context) {
...
+	}
+
+	c.JSON(http.StatusOK, areas)
+}
+
+// ParseStarKey splits key into userID and vendorID
+func ParseCuisineTypeKey(key string) (*database.CuisineTypes, error) {
+	const uuidLength = 45
+
+	if len(key) != uuidLength {
```
[the remark this answers is not in the document]
Comment (Wenhuan2516) [reply backend/api.go:805]: I see. 36 + 45 = 81. Thank you!

File: backend/database/database.go (hunk 2 above)
Comment (Wenhuan2516) [reply backend/database/database.go]: Sorry, I missed this one. Just deleted the extra comma. Sorry for my delay.

**Review by MakotoE — APPROVED** [review]

**Pull request merged**